scylla/benches/benchmark.rs

```diff
@@ -40,23 +40,49 @@ fn types_benchmark(c: &mut Criterion) {
 }
 
 fn calculate_token_bench(c: &mut Criterion) {
-    let simple_pk = ("I'm prepared!!!",);
-    let serialized_simple_pk = simple_pk.serialized().unwrap().into_owned();
-    let simple_pk_long_column = (
-        17_i32,
-        16_i32,
-        String::from_iter(std::iter::repeat('.').take(2000)),
-    );
-    let serialized_simple_pk_long_column = simple_pk_long_column.serialized().unwrap().into_owned();
+    let mut serialized_simple_pk = NewSerializedValues::new();
+    serialized_simple_pk
+        .add_value(&"I'm prepared!!!", &ColumnType::Text)
+        .unwrap();
 
-    let complex_pk = (17_i32, 16_i32, "I'm prepared!!!");
-    let serialized_complex_pk = complex_pk.serialized().unwrap().into_owned();
-    let complex_pk_long_column = (
-        17_i32,
-        16_i32,
-        String::from_iter(std::iter::repeat('.').take(2000)),
-    );
-    let serialized_values_long_column = complex_pk_long_column.serialized().unwrap().into_owned();
+    let mut serialized_simple_pk_long_column = NewSerializedValues::new();
+    serialized_simple_pk_long_column
+        .add_value(&17_i32, &ColumnType::Int)
+        .unwrap();
+    serialized_simple_pk_long_column
+        .add_value(&16_i32, &ColumnType::Int)
+        .unwrap();
+    serialized_simple_pk_long_column
+        .add_value(
+            &String::from_iter(std::iter::repeat('.').take(2000)),
+            &ColumnType::Text,
+        )
+        .unwrap();
```

Comment:
Instead of adding values one by one, why not use `NewSerializedValues::from_serializable()` and pass a tuple to it, similarly to how it is done now? It should make it shorter.

Reply:
Because then we need serialization context, which is created from prepared statement - and this benchmark doesn't even connect to Scylla as far as I can tell, so I have no way to create a prepared statement.

Reply:
I could add a way to craft context manually, but I doubt the resulting benchmark code would be simpler / shorter.

Reply:
Ok, I see, in that case I agree it's not worth it. Creating a context manually would require adding more irrelevant details compared to what we have now (e.g. column names).